### PR TITLE
Add Bootstrap 5 to the docs and playground (closes #913).

### LINKED ISSUE #913: Add Bootstrap 5 to the docs and playground
As said in https://github.com/vazco/uniforms/issues/874#issuecomment-805732210, we need to:
* Update the docs with the Bootstrap 5 theme.
  * [x] `README.md`.
  * [x] Fields API.
* Update the website:
  * [x] Theme picker in the examples.
    * We could replace our custom tabs with the ones [built-in Docuasurus](https://docusaurus.io/docs/markdown-features/tabs).
  * [x] Comparison matrix.
  * [x] Running text (e.g., "What are uniforms?").
  * [x] Playground.

### PR DESCRIPTION
**Pull request by Monteth**
This PR introduces Bootstrap5 theme documentation for uniforms (#874, #913).
It includes: 
- Readme
- Fields API
- Theme pickers
- Comparison matrixes
- Image in the bridge documentation
- Texts in index page and documentation
- Playground
- Enabled Bootstrap5 styles on the website

@radekmie proposed using the Docusaurus tabs in (#913), but as I didn't notice a significant improvement but a few obstacles, I chose not to use this solution now.

### CODE REVIEW

File: website/lib/styles.tsx [hunk 1]
```diff
@@ -36,6 +36,11 @@ const styles = {
     'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/css/bootstrap.min.css',
   ]),
 
+  bootstrap5: style([
+    'https://cdnjs.cloudflare.com/ajax/libs/octicons/3.5.0/octicons.min.css',
+    'https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css',
```
Comment (Monteth) [inline website/lib/styles.tsx:41]: https://getbootstrap.com/docs/5.0/getting-started/download/#cdn-via-jsdelivr

File: docs/api-fields.md [hunk 2]
```diff
@@ -88,23 +88,23 @@ A checkbox.
```
Comment (Monteth) [inline docs/api-fields.md:88]: As reading this file in this form hurts your eyes, take a look at least on 'feedbackable', 'inline' and 'labelBefore' props, as  have not found sufficient sources on this subject.

File: website/lib/styles.tsx (hunk 1 above)
Comment (radekmie) [reply website/lib/styles.tsx:41]: As I can see, [it's available on cdnjs as well](https://cdnjs.com/libraries/twitter-bootstrap/5.0.0-beta3). Could we use this one instead?

File: docs/api-fields.md [hunk 3]
```diff
@@ -88,23 +88,23 @@ A checkbox.
...
+|      `grid`       |        Field layout. Bootstrap grid layout style. Number is an equivalent of {sm: n}. Object is a {mode: size} object. Complete string is simply passed through.        |        bootstrap3, bootstrap4, bootstrap5         |
+|      `help`       |                                                           Help text. _Some description would be great, huh?_                                                            |     antd, bootstrap3, bootstrap4, bootstrap5      |
+|  `helpClassName`  |                                                      Help block className. _Some description would be great, huh?_                                                      |        bootstrap3, bootstrap4, bootstrap5         |
+|     `inline`      |                      Checkbox inline state. In bootstrap themes, a label is rendered as a text but in inline mode, it's treated as a field label.                       |         bootstrap3, bootstrap4,bootstrap5         |
```
Comment (radekmie) [inline docs/api-fields.md:99]: Here, and a few times later, you've missed a space before the comma.

File: docs/api-fields.md [hunk 4]
```diff
@@ -88,23 +88,23 @@ A checkbox.
...
+| :---------------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----------------------------------------------: |
+|   `appearance`    |                 Field appearance. Set to "toggle" to appear as a Material Toggle or to "checkbox" (or leave it undefined) to use a Checkbox appearance.                 |                     material                      |
+|      `extra`      |                                 Extra feedback text. In the antd theme, this renders addtional help text below any validation messages.                                 |                       antd                        |
+|  `feedbackable`   |                                                      Field feedback state. _Some description would be great, huh?_                                                      |                    bootstrap4                     |
```
Comment (radekmie) [inline docs/api-fields.md:95]: Actually, it looks like it's available only in `uniforms-bootstrap4`.

File: reproductions/package.json [hunk 5]
```diff
@@ -18,6 +18,7 @@
     "uniforms-antd": "^3.0.0",
     "uniforms-bootstrap3": "^3.0.0",
     "uniforms-bootstrap4": "^3.0.0",
+    "uniforms-bootstrap5": "^3.0.0",
```
Comment (radekmie) [inline reproductions/package.json:21]: Update the `package-lock.json` as well.

**Review by radekmie — CHANGES_REQUESTED** [review]
Have you looked into using Docusaurus tabs?

**Review by radekmie — APPROVED** [review]

**Comment by radekmie** [comment]
@Monteth Could you add a `style` in the reproductions app as well?

**Review by radekmie — APPROVED** [review]

**Pull request merged**